src/buildfile

```diff
@@ -2,23 +2,22 @@ intf_libs = # Interface dependencies.
 impl_libs = # Implementation dependencies.
 #import impl_libs += libhello%lib{hello}
 
-./: lib{benchmark} lib{benchmark_main}
+./: lib{benchmark} liba{benchmark_main}
 
 pub = [dir_path] ../include
 include $pub
 pub_hdrs = $pub/{$pub_hdrs}
 
-libul{benchmark}: cxx{* -benchmark_main} hxx{*} $pub/$pub_hdrs
-lib{benchmark}: libul{benchmark}
-lib{benchmark_main}: cxx{benchmark_main} libul{benchmark}
+lib{benchmark}: cxx{* -benchmark_main} hxx{*} $pub/$pub_hdrs
+liba{benchmark_main}: cxx{benchmark_main} lib{benchmark}
 
 cxx.poptions =+ "-I$out_root/include" "-I$src_root/include" 
 
-windows_additional_defines = 
-if($cxx.target.system == 'win32-msvc')
-  windows_additional_defines = '-D_CRT_SECURE_NO_WARNINGS'
+# windows_additional_defines = 
+# if($cxx.target.system == 'win32-msvc')
+#   windows_additional_defines = '-D_CRT_SECURE_NO_WARNINGS'
 
-cxx.poptions += $windows_additional_defines
+# cxx.poptions += $windows_additional_defines
```

Comment:
There is nothing wrong with disabling this while building the library itself so I would uncomment this if it helps.